lib/generator/models.ex

```diff
@@ -170,4 +170,25 @@ defmodule Thrift.Generator.Models do
     end
   end
 
+  def to_thrift(base_type, _file_group) when is_atom(base_type) do
+    Atom.to_string(base_type)
+  end
+  def to_thrift({:map, {key_type, val_type}}, file_group) do
+    "map<#{to_thrift key_type, file_group},#{to_thrift val_type, file_group}>"
+  end
+  def to_thrift({:set, element_type}, file_group) do
+    "set<#{to_thrift element_type, file_group}>"
+  end
+  def to_thrift({:list, element_type}, file_group) do
+    "list<#{to_thrift element_type, file_group}>"
+  end
+  def to_thrift(%Thrift.Parser.Models.TEnum{name: name}, _file_group) do
+    "#{name}"
```

Comment:
Can't this just be `name` (and below, as well)? That's preferable to building a new string object as a result of the interpolation.